plugins/parsers/prometheus/parser_test.go

```diff
@@ -0,0 +1,137 @@
+package prometheus
+
+import (
+	"testing"
+
+	"github.com/influxdata/telegraf"
+
+	"github.com/stretchr/testify/assert"
+)
+
+const (
+	validUniqueGauge = `# HELP cadvisor_version_info A metric with a constant '1' value labeled by kernel version, OS version, docker version, cadvisor version & cadvisor revision.
+# TYPE cadvisor_version_info gauge
+cadvisor_version_info{cadvisorRevision="",cadvisorVersion="",dockerVersion="1.8.2",kernelVersion="3.10.0-229.20.1.el7.x86_64",osVersion="CentOS Linux 7 (Core)"} 1
+`
+	validUniqueCounter = `# HELP get_token_fail_count Counter of failed Token() requests to the alternate token source
+# TYPE get_token_fail_count counter
+get_token_fail_count 0
+`
+
+	validUniqueLine = `# HELP get_token_fail_count Counter of failed Token() requests to the alternate token source
+`
+
+	validUniqueSummary = `# HELP http_request_duration_microseconds The HTTP request latencies in microseconds.
+# TYPE http_request_duration_microseconds summary
+http_request_duration_microseconds{handler="prometheus",quantile="0.5"} 552048.506
```

Comment:
Should we add some tests for parsing metrics with timestamps?

Reply:
If you have some scenarios, please, share and I can add them to the suit

Reply:
@DmitrySenin I happened to be looking at the prometheus docs and realized there are examples of this ask there. [Here](https://prometheus.io/docs/instrumenting/exposition_formats/#text-format-example) you go.

Reply:
Oh, got you. I was thinking that the ask was about "timestamp" values in labels. Sure, I will add new cases then

Reply:
Ops, I didn't read the docs so thoroughly. Thanks for point this out. Added a test for a counter with timestamp